website/docs/cli.md

```diff
@@ -98,6 +98,160 @@ against the prefix of these relative paths; others are substring matches.
 Matches must be against whole component parts, so `foo` matches `/foo/bar.rb`
 and `/bar/foo/baz.rb` but not `/foo.rb` or `/foo2/bar.rb`.
 
+## Accepting autocorrect suggestions
+
+From time to time, Sorbet suggests autocorrects that can be accepted to
+automatically fix certain errors. For example:
```

Comment:
```suggestion
For certain errors, Sorbet suggests autocorrects that can be accepted to
automatically fix said errors. For example:
```

I don't know what the right phrasing is, but "From time to time" evokes Sorbet doing things on a whim for me.